packages/stores/src/queries/lockup/account-locked.ts

```diff
@@ -407,6 +407,14 @@ export class ObservableQueryAccountLockedInner extends ObservableChainQuery<Acco
       });
     }
   );
+
+  readonly getPeriodLockById = computedFn((lockId) => {
+    if (!this.response) {
+      return undefined;
```

Comment:
Have you observed any problems with this? Previous state issues occurred because observables had an initial `undefined` value. I'm not sure if this would also be related with `computedFn`

Reply:
I think it's fine with computeds. It will still register with the response to listen for changes for a given `lockId`.